virttest/utils_libvirt/libvirt_misc.py

```diff
@@ -0,0 +1,28 @@
+"""
+Virsh command related utility functions which do not belong to special
+or single category.
+"""
+
+
+import re
+import logging
+
+
+def parse_virsh_cmd_result(func_name, vm_name, pattern=r'(\d+) +(\S+)'):
+    """
+    Execute specified virsh command and return parsed result
+
+    :param func_name: The virsh command function to be executed
+    :param vm_name: str, The vm name
+    :param pattern: str, regex for parsing the command output
+    :return: dict, the command result
+    """
+
+    ret = func_name(vm_name, ignore_status=False, debug=True)
```

Comment:
In some cases, we may need some special options for the command and need to check the exist status of this command, so how about just parsing the command's output? I mean, the command is executed in other places, and the stdout will be parsed by this function. What do you think?

Reply:
1. Originally I want this function can provide a single entry to execute some virsh functions, and the user only needs to provide a virsh function name to invoke it. That is a simple usage.
2. This function is not designed for those commands which the user needs to manually check their execution results, so `ignore_status=False,` is explicitly defined. 
3. This function is not designed for those commands which need more options. So yes, it has limitations.
4. This is only designed for certain virsh commands which have similar output formats. If it is extended to bigger scope, I am not sure how to write the following list parsing codes, like     `  info_dict[info[0]] = info[1]` is hard to meet other scenarios.
@Yingshun

Reply:
updated